schema_based_on_diagram.sql

```diff
@@ -0,0 +1,49 @@
+//Database schema based on a given diagram
+
+CREATE TABLE patients (
+    id serial primary key NOT NULL,
+    name varchar,
+    date_of_birth date
+);
+
+CREATE TABLE medical_histories (
+    id serial primary key NOT NULL,
+    admitted_at timestamp,
+    patient_id int,
+    status varchar ,
+    PRIMARY KEY(id),
+    CONSTRAINT fk_patient,
+      FOREIGN KEY(id) ,
+	  REFERENCES patient(patient_id)
+);
+
+CREATE TABLE treatments (
+    id serial primary key NOT NULL,
+    type varchar,
+    name varchar 
+    CONSTRAINT fk_patient
+    FOREIGN KEY(id) REFERENCES medical_histories(id)
+);
+
+CREATE TABLE invoices (
+    id serial primary key NOT NULL,
+    total_amount decimal,
+    generated_at timestamp,
+    payed_at timestamp,
+    medical_history_id int
+    date_of_birth date 
```

Comment:
- [ ] Checking through the provided ERD, this field is not supposed to be here.

![image](https://user-images.githubusercontent.com/86318284/207370498-43182a37-91de-48e0-98cf-ae9e8091d10f.png)

Reply:
Hello Nemwel-Boniface,
Good evening. Please, the above table is there among other tables provided.
Thank you.
Emem Ekpo

Reply:
Oh, my apologies @ecekpo . I meant the date of birth field. Good job for making the changes